src/main/java/net/dries007/tfc/world/surface/builder/SaltFlatsSurfaceBuilder.java

```diff
@@ -0,0 +1,26 @@
+package net.dries007.tfc.world.surface.builder;
+
+import net.dries007.tfc.world.noise.Noise2D;
+import net.dries007.tfc.world.noise.OpenSimplex2D;
+import net.dries007.tfc.world.surface.SurfaceBuilderContext;
+import net.dries007.tfc.world.surface.SurfaceStates;
+
+public class SaltFlatsSurfaceBuilder implements SurfaceBuilder
+{
+    public static final SurfaceBuilderFactory INSTANCE = SaltFlatsSurfaceBuilder::new;
+
+    public SaltFlatsSurfaceBuilder(long seed) {}
+
+    @Override
+    public void buildSurface(SurfaceBuilderContext context, int startY, int endY)
+    {
+        if (context.getBlockState(66).canBeReplaced())
```

Comment:
Same here

Reply:
No really sure what you meant by this comment because I'm not modifying the slope here, but I have switched this to check the StartY instead of checking if a block is air/replaceable